qemu/tests/multi_disk.py

```diff
@@ -106,23 +108,41 @@ def _do_post_cmd(session):
             session.cmd_status_output(cmd)
         session.close()
 
-    def _get_disk_index(session, image_size, disk_indexs):
-        list_disk_cmd = "echo list disk > disk && "
-        list_disk_cmd += "echo exit >> disk && diskpart /s disk"
+    def _get_disk_index(session, image_size, black_list, disk_indexs):
+        if params.get("os_type") == "linux":
+            list_disk_cmd = "lsblk -oKNAME,MOUNTPOINT,SIZE,MODEL"
```

Comment:
you can list_disk_cmd with params.get("list_disk_cmd") and set the value for linux and windows guest